basic-data-structures/copy-an-array-with-the-spread-operator.js

```diff
@@ -0,0 +1,14 @@
+function copyMachine(arr, num) {
+  let newArr = [];
+  // Only change code below this line
+  for (let i = 0; i < arr.length; i++) {
```

Comment:
This loop doen't make sense. You only need to define the empty newArr once with let newArr = []
With this loop you are doing nothing but [] = [] "num" times.